docs/development.md

```diff
@@ -65,9 +65,18 @@ notes and other details are available on GitHub](https://github.com/trinodb/trin
 
 ## Release process
 
-A full release process consists of the following steps:
+A full release process consists a number of steps:
 
-Merge the pull request for the release notes and pull the changes locally:
+Create a release notes pull request including the following changes:
+
+- Add new release notes in `docs/release-notes.md`.
+- Update version in `docs/quickstart.md`.
+- Update `appVersion` to new version and `version` to new version with `.0.0`
+  appended in `helm/trino-gateway/Chart.yaml`
```

Comment:
Maybe? not sure.
```suggestion
- Update `appVersion` to new version and `version` to `<version>.0.0`
  in `helm/trino-gateway/Chart.yaml`
```

Reply:
No .. I think thats less explicit. Added an example.